winterfell/webapp/src/js/profileCtrl.js

```diff
@@ -15,32 +15,15 @@ app.controller('profileCtrl', ['$scope', '$location', '$window', 'Profile', 'cla
 
     function init() {
       for (var i = 0; i < $scope.claims.length; i++) {
-        $scope.claims[i].id = $scope.claims[i].externalId; // todo, server shouldn't be sending rowId and externalId. We only reference claims through claim.id
+        $scope.claims[i].id = $scope.claims[i]._id;
         $scope.claims[i].header = createHeaderString($scope.claims[i]);
         $scope.claims[i].date = $filter('date')(new Date($scope.claims[i].stateUpdatedAt), 'MM/dd/yyyy');
       }
     }
     init();
 
     $scope.clickEditInfo = function() {
-      var newScope = $scope.$new(true);
```

Comment:
this seems like git error I dont know why I would have changed this, sorry will fix